pkg/query/flamegraph_arrow.go

```diff
@@ -691,6 +698,7 @@ func newFlamegraphBuilder(
 
 		// ensuring that we always have space to set the first row below
 		children:       make([]map[uint64]int, maxInt64(rows, 1)),
+		childrenList:   make([][]int, maxInt64(rows, 1)),
```

Comment:
```suggestion
		childrenList:   make([][]int, max(rows, 1)),
```

Reply:
We already have a max function defined that expects `a` and `b` to be of type `int`. We can refactor to use the 1.21 built-ins but here I would stick with what we used in the line above.